src/utils/internal/validate.ts

```diff
@@ -38,7 +38,10 @@ export async function validateData<T>(
 function createValidationError(validateError?: any) {
   throw createError({
     status: 400,
-    message: validateError.message || "Validation Failed",
-    ...validateError,
+    statusMessage: "Bad Request",
+    message: validateError.message || "Bad Request",
+    data: {
+      ...validateError,
```

Comment:
We might directly pass error to `data` or `cause` without destructure or use a known specific key as it might be unsafe on artibrary error objects

Reply:
As you prefer, as long as we have the data :)